docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+
+services:
+  app:
+    build:
+      context: .
+    ports:
+      - "8000:8000"
+    command: >
+      sh -c "python manage.py migrate &&
+            python manage.py wait_for_db &&
+            python manage.py runserver 0.0.0.0:8000"
```

Comment:
Change commands order, wait_for_db should be the first, because we can do migration only after that our db will be ready